events/mbed_events.h

```diff
@@ -23,8 +23,7 @@
 
 #include "events/mbed_shared_queues.h"
 
-/** \ingroup mbed-os-internal */
-/** \addtogroup events-internal-api */
+/** \addtogroup events-public-api */
```

Comment:
That's 180deg change, why?

Reply:
The file is not in `events/internal` but `events/` that's where the `events` Public headers are located.